src/components/buttons/pages/buttonsReact.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ButtonPrimary from '../ButtonPrimary.jsx'
+
+const Buttons = () => {
+    return (
+        <div>
+            <ButtonPrimary type='dark'>jakis text !!!!!</ButtonPrimary>
```

Comment:
English please 🇬🇧 Also, make sure that styleguide looks exactly the same after this PR.